scalableminds/webknossos
# Support new elementclass Float32x16

### Steps to test:
- create 16-channel float32-dataset
```
import wkw
import numpy as np

with wkw.Dataset.open(
    "./float32x16-layer", wkw.Header(voxel_type=np.float32, num_channels=16, file_len=1)
) as ds:
    data = np.zeros((16, 100, 100, 100), dtype=np.float32)
    data[0, 0:50, 0:50, 0:50] = 100
    data[1, 0:50, 0:50, 0:50] = 50
    data[2, 0:50, 0:50, 0:50] = 10
    data[10, 0:50, 0:50, 0:50] = 12
    ds.write((0, 0, 0), data)

```
write some data into it with 
`ds.write(np.array([0,0,0]), data)`
(data needs to be four dimensions of np.float32 with the first having length 16)


click “import” in wkw for this dataset, should work without errors
